css/style.css

```diff
@@ -0,0 +1,439 @@
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+
+.clear {
+  clear: both;
+}
+
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+     margin: auto;
+     color: #000;
+     font-size: 12px;
+     line-height: 1.4;
+     font-family: 'Economica', sans-serif;
+     background-image: url("../images/bknd2b.jpg");
+     background-size: cover;
+     background-repeat: no-repeat;
+     background-color: navy;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+.container {
+  width: 960px;
+  margin: auto;
+}
+
+
+header {
+  background: #0a1629;
+  padding: 5px;
+}
+
+
+.navigation {
+  height: 30px;
+  font-size: 14px;
+  font-weight: 700;
+  color: white;
+}
+
+
+.navigation ul {
+  padding: 4px 5px 0px 0px;
+}
+
+
+.navigation li {
+  display: inline;
+}
+
+
+.navigation a {
+  text-decoration: none;
+  color: white;
+  padding: 8px;
+}
+
+
+/******************************************
+/* home page
+/*******************************************/
+
+.main-search {
+  padding-top: 200px;
+}
+
+
+.titleTxt {
+  text-align: center;
+  font-weight: 700;
+  color: white;
+  font-size: 72px;
```

Comment:
Now that we've learned about responsive font units, try converting these pixel values to ems or rems in the next draft.